charts/lagoon-core/values.yaml

```diff
@@ -465,6 +465,24 @@ backupHandler:
     targetCPUUtilizationPercentage: 80
     # targetMemoryUtilizationPercentage: 80
 
+insightsHandler:
+  enabled: true
+  replicaCount: 2
+  image:
+    repository: uselagoon/insights-handler
+    pullPolicy: Always
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: ""
```

Comment:
```suggestion
    repository: testlagoon/insights-handler
    pullPolicy: Always
    # Overrides the image tag whose default is the chart appVersion.
    tag: "pr-2997"
```

Reply:
If we do it this way, we can test that this works, and then once the corresponding Lagoon release has come out, we can revert this commit